BatalanVlad/src/PaooGame/Game.java

```diff
@@ -15,26 +15,67 @@
 import java.awt.image.BufferStrategy;
 import java.util.ArrayList;
 
+
+/**
+ * The main instance of the game
+ */
 public class Game implements Runnable
 {
-    private GameWindow      wnd;        /*!< Fereastra in care se va desena tabla jocului*/
-    private boolean         runState;   /*!< Flag ce starea firului de executie.*/
+    /**
+     * The main window of the game.
+     */
+    private GameWindow      wnd;
+
```

Comment:
Make names suggestive, you have recommandations from IDE, use them, rename wnd to gameWindow. Make code easy to read

Reply:
Refactored to gameWindow